scheduler/src/cook/prometheus_metrics.clj

```diff
@@ -44,6 +44,40 @@
 (def scheduler-filter-offensive-jobs-duration :cook/scheduler-filter-offensive-jobs-duration-seconds)
 (def scheduler-handle-status-update-duaration :cook/scheduler-handle-status-update-duaration-seconds)
 (def scheduler-handle-framework-message-duration :cook/scheduler-handle-framework-message-duration-seconds)
+;; For user resource metrics, we access them by state + resource type at runtime, so it is
+;; easier to define them all in a map instead of separate vars.
+(def user-state-count-metric-map
+  {"total" :cook/scheduler-users-total-count
+   "starved" :cook/scheduler-users-starved-count
+   "waiting-under-quota" :cook/scheduler-users-waiting-under-quota-count
+   "hungry" :cook/scheduler-users-hungry-count
+   "satisfied" :cook/scheduler-users-satisfied-count})
```

Comment:
Not sure if the codahale metrics section has definitions on the states, but can we define them in the new prom stuff? Hungry/Starved/Satisfied isn't enough for me.